slang.h

```diff
@@ -536,8 +536,8 @@ extern "C"
         SLANG_CONTAINER_FORMAT_SLANG_MODULE,
     };
 
-    typedef int SlangPassThrough;
-    enum
+    typedef int SlangPassThroughIntegral;
+    enum SlangPassThrough : SlangPassThroughIntegral
```

Comment:
I’m not sure about the use of `Integral` here. First, “integral” to me strongly implies a relationship to the integral calculus, rather than an “integer” (this may be a US vs. UK English difference in connotation).

Second, it seems like this is such a closely-related type that we shouldn’t have such a distinct name. Maybe `SlangPassThrough` and `_SlangPassThrough`?

Reply:
This change is also technically source-breaking for any client that used `SlangPassThrough` in ways that are valid for `int` but not an `enum`.